frameworks/g-cloud-7/messages/dates.yml

```diff
@@ -1,4 +1,5 @@
-clarifications_close_date: "5pm, 22 September 2015"
-clarifications_publish_date: "5pm, 29 September 2015"
-framework_close_date: "3pm, 6 October 2015"
-framework_live_date: "23 November 2015"
+clarifications_close_date: '5pm <abbr title="Greenwich Mean Time">GMT</abbr>, 22 September 2015'
```

Comment:
Talking to @superroz, we're out of BST and back into GMT now so this is actually correct.

For reference: http://www.timeanddate.com/time/united-kingdom-bst.html

Note: I wish we were still in BST :(